test/config/test_section.rb

```diff
@@ -106,6 +106,42 @@ class TestSection < ::Test::Unit::TestCase
             assert_equal("8", s.num3)
           end
         end
+
+        sub_test_case '#to_s' do
+          test '#to_s == #inspect' do
+            h1 = {name: "s1", num: 10, class: "A"}
+            s1 = Fluent::Config::Section.new(h1)
+
+            assert_equal(s1.to_s, s1.inspect)
+          end
+        end
+
+        data("inspect": [:inspect, true],
```

Comment:
{"string": value} JSON syntax has been introduced in Ruby 2.3.
But Fluentd v0.14 targets Ruby 2.1 or later.
We should use {"string" => value} JSON syntax.